gax-grpc/src/main/java/com/google/api/gax/grpc/GrpcCallContext.java

```diff
@@ -149,20 +150,20 @@ public GrpcCallContext withTimeout(Duration rpcTimeout) {
   }
 
   @Override
-  public GrpcCallContext withStreamWaitTimeout(Duration streamWaitTimeout) {
-    if (streamWaitTimeout != null) {
-      Preconditions.checkArgument(
-          streamWaitTimeout.compareTo(Duration.ZERO) > 0, "Invalid timeout: <= 0 s");
-    }
+  public GrpcCallContext withStreamWaitTimeout(@Nonnull Duration streamWaitTimeout) {
```

Comment:
Actually... if it's legal for the stream wait timeout and stream idle timeout to be null, we need to make it possible for users to "null out" values if they have been set previously.